package.json

```diff
@@ -67,7 +67,7 @@
 		"lodash": "=4.17.4",
 		"method-override": "=2.3.10",
 		"pg-monitor": "=0.9.0",
-		"pg-promise": "=8.1.0",
+		"pg-promise": "=8.2.1",
```

Comment:
@vitaly-t There is a seperate PR you created for this upgrade. 
https://github.com/LiskHQ/lisk/pull/1718
It will create conflicts on merge, should not you base current PR to the one with upgrade.

Reply:
I couldn't do this PR without upgrading the dependency.